service/src/test/java/org/jboss/sbomer/service/test/integ/feature/sbom/SBOMResourceRSQIT.java

```diff
@@ -162,10 +166,7 @@ void testFetchSbomByPurlNotFound() {
                         .statusCode(404)
                         .body("errorId", CoreMatchers.isA(String.class))
                         .body("error", CoreMatchers.is("Not Found"))
-                        .body(
-                                "message",
-                                CoreMatchers.is(
-                                        "SBOM with purl = 'pkg:maven/org.apache.logging.log4j/log4j@2.19.0.redhat-00001?type=pom' couldn't be found"))
+                        .body("message", CoreMatchers.is("SBOM with purl = '" + PURL + "' couldn't be found"))
```

Comment:
I wouldn't change the expectation to be a variable. It's better to have strings, even if we are repeating these.